### PR TITLE
CocoaPods support

### PR DESCRIPTION
**Pull request by Brett-Best**
Added a starting pod spec, would be great if you could check it.

### CODE REVIEW

**Comment by CodeSlicing** [comment]
Hey, thanks for the contribution. I don't have time to review right now (although on first glance it looks good) since I'm currently in the middle of trying to release v1.13.0 which is a huge update. I can take a look at the weekend though. 

I'll also have to brush up on my pod spec knowledge. It's been a while ;o)

**Comment by CodeSlicing** [comment]
Hi there,

Over the next couple of days, I'm actually going to have enough time to look at this! I know it's been ages, but I've been caught up in a couple of really big updates and I'm about to release the last of the two. So hopefully tomorrow I can have a look at the pod file and you and we can get it merged.

**Comment by Brett-Best** [comment]
No worries, thanks!

File: PureSwiftUI.podspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |spec|
+  spec.name         = "PureSwiftUI"
+  spec.version      = "1.13.2"
```
Comment (CodeSlicing) [inline PureSwiftUI.podspec]: Right, let's get the version updated to 1.14.0 and then I'll approve the PR.

**Review by CodeSlicing — CHANGES_REQUESTED** [review]
This looks good. I've tested that the changes work as a podspec (even though it's not going to be on the cocoapods Specs repo for a while, the direct link to a tag in the podfile is working fine) - I'll update the readme to include installation instructions tomorrow. Cheers.

File: PureSwiftUI.podspec (hunk 1 above)
Comment (Brett-Best) [reply PureSwiftUI.podspec]: Updated the version to `1.14.1`.

**Comment by Brett-Best** [comment]
That’s great, thanks!

**Pull request merged**